internal/cli/cli.go

```diff
@@ -132,30 +150,53 @@ func createServicesCommand(controller controller.ControllerInterface) *cobra.Com
 			controller.Services(args...)
 		},
 	}
+
+	svsCmd.AddCommand(createServiceAccessCommand(controller))
+	return svsCmd
 }
 
-func createTeleportCommand(controller controller.ControllerInterface) *cobra.Command {
-	teleportCmd := &cobra.Command{
-		Use:   "teleport",
+func createServiceAccessCommand(controller controller.ControllerInterface) *cobra.Command {
+	return &cobra.Command{
+		Use:     "access",
+		Aliases: []string{"envs", "environment", "environments"},
+		Short:   "List all the Environment IDs and their regions",
+		Long:    `Get a list of all the Environment ID names and their regions`,
+		Args:    cobra.ExactArgs(2),
+		Run: func(cmd *cobra.Command, args []string) {
+			controller.ServiceAccess(args[0], args[1])
+		},
+	}
+}
+
+func createAccessCommand(controller controller.ControllerInterface) *cobra.Command {
+	accessCmd := &cobra.Command{
+		Use:   "access",
 		Short: "Port-forward your remote services to your local machine",
 		Run: func(cmd *cobra.Command, args []string) {
-			teleportAllFlag, _ := cmd.Flags().GetBool("all")
-			controller.Teleport(teleportAllFlag)
+			controller.Access()
 		},
 	}
-	teleportCmd.Flags().Bool("all", false,
-		"Allow accessing all envs without .git")
+	return accessCmd
+}
 
-	return teleportCmd
+func createTeleportCommand(controller controller.ControllerInterface) *cobra.Command {
+	accessCmd := &cobra.Command{
+		Use:   "teleport",
+		Short: "Teleport into your remote services",
+		Run: func(cmd *cobra.Command, args []string) {
```

Comment:
comment - might need a better description cuz it's actually port-forwarding all the services in an environment and "teleporting" one backend api.